static/src/javascripts/projects/common/utils/$.js

```diff
@@ -0,0 +1,33 @@
+import bonzo from 'bonzo';
+import qwery from 'qwery';
+
+// Warning: side effect. This patches the bonzo module for use everywhere
+bonzo.aug({
+    height() {
+        return this.dim().height;
+    },
+});
+
+function $(selector, context) {
+    return bonzo(qwery(selector, context));
+}
+
+$.create = s => bonzo(bonzo.create(s));
+
+$.ancestor = (el, c) => {
+    if (el.nodeName.toLowerCase() === 'html') {
+        return false;
+    }
+    if (!el.parentNode || bonzo(el.parentNode).hasClass(c)) {
+        return el.parentNode;
+    }
+    return $.ancestor(el.parentNode, c);
+};
+
+$.forEachElement = (selector, fn) => {
+    const els = qwery(selector);
+    els.forEach(fn);
+    return els;
+};
```

Comment:
it does throw up an interesting question with regards to how much we want to revisit code as we convert it. i guess that would have to be up to the individual, since it's their PR and their responsibility to deal with any errors it introduces etc. 

there's some good points in this conversation though, but i wonder if they would be better addressed as separate PRs?